### PR TITLE
Improving ux

### PR DESCRIPTION
**Pull request by bumblebot1**
Added some of the front end tests will add some more over the next pull requests

### CODE REVIEW

File: src/client/game/answer-input.js [hunk 1]
```diff
@@ -229,7 +229,7 @@ export class AnswerInput extends Component {
 
 const mapStateToProps = state => ({
     answerTimerValue: state.game.answerTimerValue,
-    submissionTimerValue: state.game.submissionTimerValue
+    submissionTimerValue: state.game.submissionTimerValue,
```
Comment (kenneth-gray) [inline src/client/game/answer-input.js]: No comma-dangle please :smile:

Comment (thullSL) [inline src/client/game/answer-input.js]: , A problem that doesn't exit with prepended commas

File: src/client/lobby/lobby.js [hunk 2]
```diff
@@ -16,11 +19,27 @@ export class Lobby extends Component {
...
+
     render() {
         return (
             <div>
                 <div className="text-center">
-                    <h1>Lobby</h1>
+                    <h1>
+                        Lobby
+                        <label className={style['set-username-label']}>
+                            <label>{'Username: '}</label>
```
Comment (thullSL) [inline src/client/lobby/lobby.js]: Is this creating a code block to return a static string? `<label>Username:</label>`

Comment (JamieZafar) [inline src/client/lobby/lobby.js]: yeah, it's to show the user where they should be putting their username

File: src/client/lobby/lobby.spec.js [hunk 3]
```diff
@@ -4,12 +4,33 @@ import { shallow } from 'enzyme';
 import { Lobby } from './lobby';
 
 describe('<Lobby />', () => {
+	let wrapper;
```
Comment (kenneth-gray) [inline src/client/lobby/lobby.spec.js]: indentation

File: src/client/lobby/lobby.js (hunk 2 above)
Comment (kenneth-gray) [inline src/client/lobby/lobby.js]: The point being made: remove the braces and just put `Username: ` in the label :smile:

Comment (thullSL) [inline src/client/lobby/lobby.js]: but why the code block...

Comment (JamieZafar) [inline src/client/lobby/lobby.js]: ah yeah i see :see_no_evil:

**Comment by thullSL** [comment]
:shipit:

File: src/client/lobby/room-creator.js [hunk 4]
```diff
@@ -7,39 +7,36 @@ import storageService from '../services/storage-service';
...
-        cloakService.messageSetUsername(this.state.username);
-        storageService.storeName(this.state.username);
-    }
 
     createRoom() {
         cloakService.messageCreateRoom(this.state.roomname);
     }
 
-    handleChange(event, property) {
+    handleChange(event) {
```
Comment (kenneth-gray) [inline src/client/lobby/room-creator.js]: Please update the name of this function now it's not generic any more

File: src/client/lobby/room-creator.spec.js [hunk 5]
```diff
@@ -4,18 +4,64 @@ import { shallow } from 'enzyme';
...
+    });
+
+    it('sends the createRoom message when the key pressed is enter', () => {
+        let component = wrapper.instance();
+
+        component.handleChange({
+            target:{value: 'testingRoom'}
+        });
+        component.handleEnterPress({
+            which: 13
```
Comment (kenneth-gray) [inline src/client/lobby/room-creator.spec.js]: Another const?

File: src/client/rooms/room-page.spec.js [hunk 6]
```diff
@@ -5,30 +5,92 @@ import { mount, shallow } from 'enzyme';
...
+            );
+
+            expect(cloakService.messageJoinRoom).toHaveBeenCalledWith(data);
+        });
+
+        it('if user is connected to the server it tries to get the room data', () => {
+            data = '1234';
+            cloakService.isConnected.and.returnValue(true);
+            let wrapper = mount(
+                <Provider store={store}>
```
Comment (kenneth-gray) [inline src/client/rooms/room-page.spec.js]: For future reference - nothing to change here - if you export the main class as well as the default export, you can then use `import { CLASS } from 'FILE';`; then you don't need to use the Provider or store any more

**Pull request merged**